src/sentry/apidocs/parameters.py

```diff
@@ -271,3 +271,46 @@ class ReplayParams:
         type=OpenApiTypes.UUID,
         description="""The ID of the replay you'd like to retrieve.""",
     )
+
+
+class IntegrationParams:
+    PROVIDER_KEY = OpenApiParameter(
+        name="providerKey",
+        location="query",
+        required=False,
+        type=str,
+        description="""Specific integration provider to filter by. For example, `slack`. See [Integrations Documentation](/product/integrations/) for an updated list of providers.""",
```

Comment:
Can you try clicking the links on your devserver? I think you might need to do `../product` but not sure. Also how would a user know that `slack` wouldn't be capitalized from that docs page alone?
```suggestion
        description="""Specific integration provider to filter by such as `slack`. See our [Integration Documentation](/product/integrations/) for an updated list of providers.""",
```

Reply:
Tested already. This works as is.